file-formats/sktd/type/zif_aff_sktd_v1.intf.abap

```diff
@@ -0,0 +1,36 @@
+INTERFACE zif_aff_sktd_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Documented Object</p>
+    "! Type and name of the documented object
+    BEGIN OF ty_sktd_documented_object,
+      "! <p class="shorttext">Object Type</p>
+      "! Type of the documented object
+      "! $required
+      type TYPE trobjtype,  " TYPE c LENGTH 4,
+      "! <p class="shorttext">Object Name</p>
+      "! Name of the documented object
+      "! $required
+      name TYPE sobj_name,  " zif_aff_types_v1=>ty_object_name_30,
```

Comment:
The interfaces shall be self-contained. Therefore, it would be great if you use the types mentioned in the comments:

```suggestion
      type TYPE c LENGTH 4,
      "! <p class="shorttext">Object Name</p>
      "! Name of the documented object
      "! $required
      name TYPE zif_aff_types_v1=>ty_object_name_30,
```